mkosi/backend.py

```diff
@@ -694,6 +694,12 @@ def run_workspace_command(
     if "workspace-command" in ARG_DEBUG:
         nspawn += ["--setenv=SYSTEMD_LOG_LEVEL=debug"]
 
+    if args.image_id:
+        nspawn += [f"--setenv={args.image_id}"]
+
+    if args.image_version:
+        nspawn += [f"--setenv={args.image_version}"]
```

Comment:
Aren't the environment variable names missing here?
```suggestion
    if args.image_id:
        nspawn += [f"--setenv=IMAGE_ID={args.image_id}"]

    if args.image_version:
        nspawn += [f"--setenv=IMAGE_VERSION={args.image_version}"]
```

Reply:
🤦‍♂️ Whoops. Fixed now.